README.md

```diff
@@ -1 +1,154 @@
-# book-stores
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Math-app\] ](#-math-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Installation and Setup](#installation-and-setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [� License ](#-license-)
+
+
+
+# 📖 [Math-app] <a name="about-project"></a>
```

Comment:
- [ ] As required by MIcroverse, your readme file needs to be descriptive and look professional, I'll suggest you remove the name `Math App`,  and update the t=name correctly as this is the `Book-store ` project👍 👍

Reply:
Great, thanks for the correction, necessary changes applied